### DISCUSSION TITLE
Your book on Go and Fiber

### DISCUSSION THREAD

**Discussion by jeex** [community/community#40687] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The Github repo you mention in your book on Go and Fiber: _go-book_ does not exist. Therefor the examples in your book do not work. Can you fix this?

**Comment by airtower-luna** [comment]
Your message reads like you're trying to send a personal message to someone, but this is a public forum. I hope you can find a way to actually reach them!
